### DISCUSSION TITLE
Server Parameters

### DISCUSSION THREAD

**Discussion by benvanderlaan1** [official-antistasi-community/A3-Antistasi#1769] [Q&A] [answered]
Hello,

I would like to ask if there is a list of all the new parameters that can be edited in a .cfg file. I am running a server with some friends and we can only change parameters through the server .cfg yet I can't seem to find how I can call the new parameters. Any help would be appreciated.

Also, if this is not the place to ask these questions feel free to remove it.

**Comment by Lazejun** [comment]
Well in General dont use Unstable, use Stable (Master Branch).
https://github.com/official-antistasi-community/A3-Antistasi/blob/master/A3-Antistasi/MissionDescription/params.hpp
These are all the Current Parameters for 2.3.2, the File will be updated when the next release is pushed.

**Comment by HakonRydland** [comment, marked as answer]
go through the classes in this file [Params.hpp](https://github.com/official-antistasi-community/A3-Antistasi/blob/master/A3-Antistasi/MissionDescription/params.hpp)
then put this in your server cfg mission cycle
```
class Missions {
      class Mission1    {
            template ="Antistasi-Malden-2-2-1.Malden"; //replace with your mission name
            difficulty = "Custom";
            class Params {
                  switchComm = 0;
                  pMarkers = 0;
                  AISkill = 2;
                  unlockItem = 40;
                  memberOnlyMagLimit = 60;
                  teamSwitchDelay = 1800;
                  .
                  .
                  .
             };
      };
};
```
where each entry in the params class is a name of a class from ``Params.hpp`` followed by `` = {value};``
the values can also be found in ``Params.hpp``
